feature/add-block-explorer-doc (#2)
yongenaelf/aelf-docs: docs/ecosystem/explorer.md

## 1. Introduction

The AElf blockchain explorer serves as a primary tool for monitoring and searching information on the main chain and its 

> side chains. Through the aelf blockchain explorer, users can:

Reviewer: ```suggestion
side chains. Through the AElf blockchain explorer, users can:
```

method called in the transaction.
- See the top 1000 wallet addresses with the most ELF tokens on the AElf chain, along with all contracts deployed on the
AElf chain.

![blockchain](/img/explorer-blockchain.png)

### 3. Viewing token symbols on AElf chain

In the top-left menu bar, there are four options. Clicking on the third option 'token' will display all token symbols on
the AElf chain, including symbols for both FT (Fungible Tokens) and NFT (Non-Fungible Tokens).

![symbol](/img/explorer-symbol.png)

### 4. Viewing Governance on AElf chain

In the top-left menu bar, there are four options. Clicking on the fourth option 'governance' reveals three sub-options: 
proposal, vote, and resource.

#### Proposal

Users can actively engage in on-chain governance by using the AElf blockchain explorer to view, initiate proposals, and 
establish their own organizations. Initiating proposals and creating organizations require logging in using a browser 
plugin.

![proposal](/img/explorer-proposal.png)

#### Vote

Users can apply to become a candidate node and vote for nodes through the AElf blockchain explorer. 
The scenario-based operations in the AElf blockchain explorer are as follows:

> (1) Applying to Become a Candidate Node

Reviewer: ```suggestion
##### (1) Applying to Become a Candidate Node
```

> Users want to become a production node on AElf, participate in block production, and maintain the orderly operation of 
> the entire AElf ecosystem. At this point, users need to apply to become a node on the AElf blockchain explorer.

Reviewer: ```suggestion
A node operator wants their node to become a production node on AElf, participate in block production, and maintain the orderly operation of the entire AElf ecosystem. At this point, the node owner needs to apply for their node to become a production node on the aelf blockchain. This can be done using the AElf blockchain explorer.
```
